mtags/src/main/scala-2/scala/meta/internal/pc/PcCollector.scala

```diff
@@ -223,11 +223,15 @@ abstract class PcCollector[T](
          * For comprehensions have two owners, one for the enumerators and one for
          * yield. This is a heuristic to find that out.
          */
-        def isForComprehensionOwner(named: NameTree) =
-          soughtNames(named.name) &&
-            named.symbol.owner.isAnonymousFunction && owners.exists(owner =>
-              pos.isDefined && owner.pos.isDefined && owner.pos.point == named.symbol.owner.pos.point
-            )
+        def isForComprehensionOwner(named: NameTree) = {
```

Comment:
It would be awesome if you could group and name some things here to make it a bit more readable.